src/NHibernate.Test/Legacy/SQLFunctionsTest.cs

```diff
@@ -76,10 +76,13 @@ public void DialectSQLFunctions()
 				rset = s.CreateQuery("select abs(round(s.Pay)) from s in class Simple").List();
 				Assert.AreEqual(46f, rset[0], "abs(round(-45.8)) result was incorrect");
 
+				/* As of NH-3893, left and right functions are broken. Seemed confused with join keyword, and not
```

Comment:
That is [NH-3893](https://nhibernate.jira.com/browse/NH-3893).

There are no other conditions than `if (false)` for this part of the test. It is broken, whatever the database. But since it was nested in an `if (Dialect is Oracle8iDialect)`, it was failing only for Oracle.